modules/client-configuration/pages/snippets/add_channels_cli_multiarch.adoc

```diff
@@ -0,0 +1,17 @@
+.Procedure: Adding Software Channels at the Command Prompt
```

Comment:
This snipped should be used for any other OS added with `spacewalk-common-channels` that support several architectures, but let's not do that change now, as I am not sure of the list, and could be that we even need to fix some stuff before that happens :-)